tests/pragmas/tcustom_pragma.nim

```diff
@@ -91,7 +96,7 @@ block: # ref types
 
   var ptrS = NodePtr(nil)
   const
-    ptrRightSerKey = getCustomPragmaVal(s.right, serializationKey)
```

Comment:
nitpick: This seems to exchange one test for another. It's better to add tests instead.